local/rest_api_gcbm/gcbm.py

```diff
@@ -259,97 +305,119 @@ def _write_to_file(file_path, data):
-    sim.set_disturbance_attributes("disturbances_2012.tiff",
+    sim.add_file("tests/tiff/new_demo_run/disturbances_2012_moja.tiff")
+    sim.set_disturbance_attributes("disturbances_2012_moja.tiff",
                                   {"year": 2012, "disturbance_type": "Wildfire", "transition": 1})
 
 
-    sim.add_file("tests/GCBM_New_Demo_Run/disturbances/disturbances_2013.tiff")
-    sim.set_disturbance_attributes("disturbances_2013.tiff",
+    sim.add_file("tests/tiff/new_demo_run/disturbances_2013_moja.tiff")
+    sim.set_disturbance_attributes("disturbances_2013_moja.tiff",
                                     {"year": 2013, "disturbance_type": "Mountain pine beetle — Very severe impact", "transition": 1})
 
     # TODO: Check how to handle multiple attributes entries (L442-451 of `app.py:master`)
-    # sim.set_disturbance_attributes("disturbances_2013.tiff",
+    # sim.set_disturbance_attributes("disturbances_2013_moja.tiff",
     #                                 {"year": 2013, "disturbance_type": "Wildfire", "transition": 1})
 
-    sim.add_file("tests/GCBM_New_Demo_Run/disturbances/disturbances_2014.tiff")
-    sim.set_disturbance_attributes("disturbances_2014.tiff",
+    sim.add_file("tests/tiff/new_demo_run/disturbances_2014_moja.tiff")
+    sim.set_disturbance_attributes("disturbances_2014_moja.tiff",
                                     {"year": 2014, "disturbance_type": "Mountain pine beetle — Very severe impact", "transition": 1})
 
-    sim.add_file("tests/GCBM_New_Demo_Run/disturbances/disturbances_2015.tiff")
-    sim.set_disturbance_attributes("disturbances_2015.tiff",
+    sim.add_file("tests/tiff/new_demo_run/disturbances_2015_moja.tiff")
+    sim.set_disturbance_attributes("disturbances_2015_moja.tiff",
                                     {"year": 2015, "disturbance_type": "Wildfire", "transition": 1})
 
-    sim.add_file("tests/GCBM_New_Demo_Run/disturbances/disturbances_2016.tiff")
-    sim.set_disturbance_attributes("disturbances_2016.tiff",
+    sim.add_file("tests/tiff/new_demo_run/disturbances_2016_moja.tiff")
+    sim.set_disturbance_attributes("disturbances_2016_moja.tiff",
                                    {"year": 2016, "disturbance_type": "Wildfire", "transition": 1})
 
-    sim.add_file("tests/GCBM_New_Demo_Run/disturbances/disturbances_2018.tiff")
-    sim.set_disturbance_attributes("disturbances_2018.tiff",
+    sim.add_file("tests/tiff/new_demo_run/disturbances_2018_moja.tiff")
+    sim.set_disturbance_attributes("disturbances_2018_moja.tiff",
                                     {"year": 2018, "disturbance_type": "Wildfire", "transition": 1})
 
     # TODO: classifiers don't have 'year' attributes
-    sim.add_file("tests/GCBM_New_Demo_Run/classifiers/Classifier1.tiff")
-    # sim.set_classifier_attributes("classifier1.tiff",
+    sim.add_file("tests/tiff/new_demo_run/Classifier1_moja.tiff", category="classifiers")
+    # sim.set_classifier_attributes("Classifier1_moja.tiff",
```

Comment:
This seems to mix to API styles - I think we want either:

```python
sim.add_file(filename, category)
sim.set_attributes(filename, category, <payload>)
```

or

```python
sim.add_<category>(filename)
sim.set_<category>_attributes(filename, payload)
```

Reply:
That's a valid concern. I added `sim.add_file(filename, category)` since it will not be expected that the category will be in the filename every time. Do you think if it's okay that we stick to the first API style?

```python
sim.add_file(filename, category)
sim.set_attributes(filename, category, payload)
```

Reply:
I agree with picking one style and sticking to it. :heavy_check_mark:

Reply:
Fixed! :)